korangar_networking/src/event.rs

```diff
@@ -118,7 +118,7 @@ pub enum NetworkEvent {
         index: InventoryIndex,
         equipped_position: EquipPosition,
     },
-    ChangeJob(AccountId, u32),
+    ChangeSprite(AccountId, SpriteChangeType, u32, u32),
```

Comment:
When you revert the `ChangeSprite` and also when adding the second change packet, it would be amazing if you could convert from a tuple variant to a struct variant (e.g. `ChangeJob { account_id: AccountId, job_id: u32 }`). I've been changing all the variants to named fields one by one when working on them.